src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export const Calculator = ()=>{
+    return <div className="container">
+    <div className="output">
+      <span>0</span>
+    </div>
+    <button type="submit">AC</button>
+    <button type="submit">+/-</button>
+    <button type="submit">%</button>
+    <button type="submit" className="operand">
+      ÷
+    </button>
+    <button type="submit">7</button>
```

Comment:
- [OPTIONAL] 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻